indexing-hadoop/src/main/java/io/druid/indexer/HadoopDruidIndexerMapper.java

```diff
@@ -104,6 +105,10 @@ public final static InputRow parseInputRow(Object value, InputRowParser parser)
     if(parser instanceof StringInputRowParser && value instanceof Text) {
       //Note: This is to ensure backward compatibility with 0.7.0 and before
       return ((StringInputRowParser) parser).parse(value.toString());
+    } else if(parser instanceof StringInputRowParser && value instanceof BytesWritable) {
```

Comment:
I would like to add this now, it's not something that a ton of people have complained about but I do think it's good to have an easy kafka -> s3 -> druid path. this patch + Secor can provide that easy path.